go/statistics.go

```diff
@@ -0,0 +1,46 @@
+package svix
+
+import (
+	"context"
+
+	"github.com/svix/svix-webhooks/go/internal/openapi"
+)
+
+type Statistics struct {
+	api *openapi.APIClient
+}
+
+type (
+	AppUsageStatsIn        = openapi.AppUsageStatsIn
+	AppUsageStatsOut       = openapi.AppUsageStatsOut
+	AggregateEventTypesOut = openapi.AggregateEventTypesOut
+)
+
+func (s *Statistics) CalculateAggregateAppStats(ctx context.Context, appUsageStatsIn *AppUsageStatsIn, options *PostOptions) (*AppUsageStatsOut, error) {
```

Comment:
```suggestion
func (s *Statistics) AggregateAppStats(ctx context.Context, appUsageStatsIn *AppUsageStatsIn, options *PostOptions) (*AppUsageStatsOut, error) {
```

Should these drop the "calculate" prefix too? Maybe we only did it for event types? I think I see the same in Java and probably others further down in the diff.

Reply:
I didn't squash the commits. Is it possible you're looking at changes from the first commit only?